addon/components/ember-notify/message.js

```diff
@@ -47,11 +47,22 @@ export default Ember.Component.extend({
   }),
   visibleObserver: Ember.observer('message.visible', function() {
     if (!this.get('message.visible')) {
-      this.send('close');
+      this.send('closeIntent');
     }
   }),
 
   actions: {
+  	// alias to close action so we can poll whether hover state is active
+  	closeIntent: function() {
+  		if (this.get('isDestroyed') || this.get('message.closed')) return;
```

Comment:
Yeah, right. This check isn't necessary because the element shouldn't be in :hover state if it's been destroyed.